src/components/Error/Error.tsx

```
@@ -0,0 +1,35 @@
+import { Stack, Typography, useMediaQuery } from '@mui/material';
+
+import { Button } from '@components';
+import { theme } from '@theme';
+
+import { ErrorProps } from './Error.type';
+
+export const Error = ({
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)